### PR TITLE
Update universe.json

### PR DESCRIPTION
**Pull request by nlptechbook**
A bot powered by Clarifai Predict API and spaCy. Can be found in Telegram messenger at @pic2phrase_bot

## Checklist

- [x ] I have submitted the spaCy Contributor Agreement.
- [x ] I ran the tests, and all new and existing tests passed.
- [x ] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Comment by nlptechbook** [comment]
The bot illustrates how Computer Vision and NLP can be efficiently used together.

**Comment by nlptechbook** [comment]
Hi,

Thank you for your reply. I've uploaded the images to Imgur, The
updated file is attached. As to the url that links to the bot, the
point is that the bots implemented on the Telegram platform do not
have urls. You need to create  a Telegram account.
Then, in the Telegram app, you can perform a search for @pic2phrase.

File: website/meta/universe.json [hunk 1]
```diff
@@ -1619,6 +1619,19 @@
             },
             "category": ["standalone", "research"]
         },
+        {
+            "id": "pic2phrase_bot",
+            "title": "pic2phrase_bot: Photo Description Generator",
+            "slogan": "A bot that generates descriptions to submitted photos, in a human-like manner.",
+            "description": "pic2phrase_bot runs inside Telegram messenger and can be used to generate a phrase describing a submitted photo, employing computer vision, web scraping, and syntactic dependency analysis powered by spaCy." 
+            "thumb": "https://drive.google.com/open?id=1GTrpPzc8j4mAmYCJZibYrADAp0GWcVHd",
+            "image": "https://drive.google.com/open?id=1t7URKJ-4uOJmZb_GbNvw-F5LLtvEoBRy",
```
Comment (nlptechbook) [inline website/meta/universe.json]: "thumb": "https://i.imgur.com/ggVI02O.jpg",
            "image": "https://i.imgur.com/z1yhWQR.jpg",

**Comment by nlptechbook** [comment]
Hi,

The following url might be used ro reach the bot:

             "url": "https://telegram.me/pic2phrase_bot"

However, as mentioned previously, before you can use it, you need to
sign in to your Telegram account.

As to the images, i switched to Imgur:

            "thumb": "https://i.imgur.com/ggVI02O.jpg",
            "image": "https://i.imgur.com/z1yhWQR.jpg",

Yuli

**Comment by svlandeg** [comment]
Hi @nlptechbook, can you make the changes in this PR by editing the file accordingly? You want to edit & push the correct file on this branch: https://github.com/nlptechbook/spaCy/tree/patch-5 and then it will automatically show up here in this PR.

**Comment by nlptechbook** [comment]
I just did it.

File: website/meta/universe.json [hunk 2]
```diff
@@ -1619,6 +1619,20 @@
             },
             "category": ["standalone", "research"]
         },
+        {
+            "id": "pic2phrase_bot",
+            "title": "pic2phrase_bot: Photo Description Generator",
+            "slogan": "A bot that generates descriptions to submitted photos, in a human-like manner.",
+            "description": "pic2phrase_bot runs inside Telegram messenger and can be used to generate a phrase describing a submitted photo, employing computer vision, web scraping, and syntactic dependency analysis powered by spaCy."
```
Comment (ines) [inline website/meta/universe.json:1626]:
```suggestion
            "description": "pic2phrase_bot runs inside Telegram messenger and can be used to generate a phrase describing a submitted photo, employing computer vision, web scraping, and syntactic dependency analysis powered by spaCy.",
```

File: website/meta/universe.json [hunk 3]
```diff
@@ -1619,6 +1619,20 @@
...
+            "slogan": "A bot that generates descriptions to submitted photos, in a human-like manner.",
+            "description": "pic2phrase_bot runs inside Telegram messenger and can be used to generate a phrase describing a submitted photo, employing computer vision, web scraping, and syntactic dependency analysis powered by spaCy." 
+            "thumb": "https://i.imgur.com/ggVI02O.jpg",
+            "image": "https://i.imgur.com/z1yhWQR.jpg",
+            "url": "https://telegram.me/pic2phrase_bot",  
+            "author": "Yuli Vasiliev",
+            "author_links": {
+                "twitter": "VasilievYuli",
+            },
+            "category": ["standalone", "research"]
```
Comment (ines) [inline website/meta/universe.json:1634]:
```suggestion
            "category": ["standalone", "conversational"]
```

**Pull request merged**